static/js/stream_pill.js

```diff
@@ -0,0 +1,51 @@
+function get_user_ids_from_sub(items) {
+    let user_ids = [];
+    const stream_ids = items.map((item) => item.stream_id);
+    for (const stream_id of stream_ids) {
+        const sub = stream_data.get_sub_by_id(stream_id);
+        if (!sub) {
+            continue;
+        }
+        const subscribers = stream_edit.get_users_from_subscribers(sub.subscribers);
+        user_ids = user_ids.concat(subscribers.map((user) => user.user_id));
```

Comment:
These two lines are weird -- `sub.subscribers` is already a list of user ids; so you want to just `user_ids = user_ids.concat(sub.subscribers)`.

Also should this be called `get_user_ids_from_sub`, since it's potentially multiple streams?